microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/commons/snippets/SnippetRegistry.java

```diff
@@ -219,7 +301,8 @@ private static MarkupContent createDocumentation(Snippet snippet, boolean canSup
 		return new MarkupContent(canSupportMarkdown ? MarkupKind.MARKDOWN : MarkupKind.PLAINTEXT, doc.toString());
 	}
 
-	private static String getInsertText(Snippet snippet, boolean replace, String lineDelimiter) {
+	private static String getInsertText(Snippet snippet, Map<String, String> model, boolean replace,

```

Comment:
I think `replace` could be named better. What gets replaced when `replace == true`?

Reply:
I tried to fix it, is it better?

Reply:
yes, this is better